app/DoctrineMigrations/Version20190709124816.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20190709124816 extends AbstractMigration
```

Comment:
This migration shouldn't have been committed, it is the same as `Version20190709120618`